docs/compute_resources.md

```diff
@@ -0,0 +1,177 @@
+memory limit, you will see the reason `OOM Killed`, as in this example:
+```
+$ kubectl get pods/memhog
+NAME      READY     REASON       RESTARTS   AGE
+memhog    0/1       OOM Killed   0          1h
+```
+*OOM* stands for Out Of Memory.
+
+A container may or may not be allowed to exceed its CPU limit for extended periods of time.
+However, it will not be killed for excessive CPU usage.
+
+## Specifying Resource Quantities
+A resource quantity (an amount of CPU or memory) can be specified as an integer,
+a floating point value, or a string.  As a string, it can optionally use an SI suffix
+(E, P, T, G, M, k, m) or a power-of-two analog (Ei, Pi, Ti, Gi, Mi, Ki).
+For example, the following JSON snippets are all valid and represent roughly the same value:
+```json
+   "cpu": 128974848,
+   "cpu": "128974848",
+   "cpu": 128974848.1,
+   "cpu": "128974848.1",
+   "cpu": 129e6,
+   "cpu": "129e6",
+   "cpu": "129M",
+   "cpu": "123Mi"
+```
+
+When an object is created or updated, the API server converts
+any resource quantities to a fixed-point representation before persisting
+the object.  It may be truncated to 3 decimal places in the process.
+All resource math is done with the fixed-point values, never floating point.
+
+
+## Monitoring Compute Resource Usage
+
+The resource usage of a pod is reported as part of the Pod status.
+
+If [optional monitoring](../cluster/addons/monitoring/README.md) is configured for your cluster,
+then pod resource usage can be retrieved from the monitoring system.
+
+## Troubleshooting
+
+If the scheduler cannot find any node where a pod can fit, then the pod will remain unscheduled
+until a place can be found.    An event will be produced each time the scheduler fails to find a
+place for the pod, like this:
+```
+$ kubectl describe pods/frontend | grep -A 3 Events
+Events:
+  FirstSeen				LastSeen			Count	From SubobjectPath	Reason			Message
+  Tue, 30 Jun 2015 09:01:41 -0700	Tue, 30 Jun 2015 09:39:27 -0700	128	{scheduler }            failedScheduling	Error scheduling: For each of these fitness predicates, pod frontend failed on at least one node: PodFitsResources.
+```
+
+If a pod or pods are pending with this message, then there are several things to try:
+- Add more nodes to the cluster.
+- Terminate unneeded pods to make room for pending pods.
+- Check that the pod is not larger than all the nodes.  For example, if all the nodes
+have a capacity of `cpu: 1`, then a pod with a limit of `cpu: 1.1` will never be scheduled.
+
+You can check node capacities with the `kubectl get nodes -o <format>` command.
+Here are some example command lines that extract just the necessary information:
+- `kubectl get nodes -o yaml | grep '\sname\|cpu\|memory'`
+- `kubectl get nodes -o json | jq '.items[] | {name: .metadata.name, cap: .status.capacity}'`
+
+The [resource quota](#docs/resource_quota_admin.md) feature can be configured
```

Comment:
I think this points to a file in the same directory? The URL should exclude '#docs' -> (./resource_quota_admin.md)